app/components/ideas.hbs

```diff
@@ -1,13 +1,13 @@
 <section>
-    {{#each model as |idea|}}
+  {{#each @model as |idea|}}
     <div class="card">
         <div class="idea-img" style="background-image: url({{idea.imageURL}});"></div>
         <div class="idea-short-description">
-            {{idea.description}}
+            {{ idea.description}}
```

Comment:
Consider install [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) extension for you VSC instance for better highlight bad code lint. Also nice to have with dev in Ember.js: [Unstable Ember Language Server](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-ember-unstable)